app/src/main/java/com/duckduckgo/app/bookmarks/ui/SaveBookmarkDialogFragment.kt

```diff
@@ -88,8 +88,8 @@ class SaveBookmarkDialogFragment : DialogFragment() {
     }
 
     private fun getExistingId(): Int? = arguments!!.getInt(KEY_BOOKMARK_ID)
-    private fun getExistingTitle(): String = arguments!!.getString(KEY_PREEXISTING_TITLE)
-    private fun getExistingUrl(): String = arguments!!.getString(KEY_PREEXISTING_URL)
+    private fun getExistingTitle(): String? = arguments!!.getString(KEY_PREEXISTING_TITLE)
+    private fun getExistingUrl(): String? = arguments!!.getString(KEY_PREEXISTING_URL)
```

Comment:
Updated to optional to match the `createDialogCreationMode` method which accepts optionals. This needs to match or a crash can occur.